app/src/test/resources/protobuf/person-widened.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package com.anentropic.chuckd.test;
+
+message Person {
+  string firstName = 1;
+  string lastName = 2;
+  int64 age = 3;
+  int64 height = 4;
+  enum Gender {
+    MALE = 0;
+    FEMALE = 1;
+    NONBINARY = 2;
+    ALIEN = 3;
```

Comment:
note that confluent doesn't care that members of the enum were added or removed, these are classified as compatible changes https://github.com/confluentinc/schema-registry/blob/064d13a7d4391990074cd884fe0d1a047738b84d/protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/diff/SchemaDiff.java#L63

to get an incompatible change we have to add/remove message types from the proto